src/chunks/storage/wrappers.rs

```diff
@@ -186,8 +186,6 @@ impl WriteChunk for StorageWrap {
         match self {
             #[cfg(all(feature = "memmap", target_endian = "little"))]
             StorageWrap::MmapArray(inner) => inner.chunk_identifier(),
-            #[cfg(target_endian = "big")]
-            StorageWrap::MmapArray(_inner) => unimplemented!(),
```

Comment:
Why is the big-endian gating removed?

Reply:
I got some warnings somewhere, probably due to the feature gating issues.

Reply:
We do not support memory mapping for big-endian architectures. It will work, but you will get incorrect embeddings, since the finalfusion format stores data in little-endian format.